app/assets/stylesheets/layouts/starry.scss

```diff
@@ -1,85 +1,217 @@
+
+/* - flashes */
+// these are the notifications that appear when you submit a form or get a message
+$bg_color_flash_error: #9692A8; // error
+$bg_color_flash_success: #9692A8; // success
+$bg_color_flash_inbox: #9692A8; // new message // TODO: THESE SHOULD NOT ALL THE SAME
+$bg_color_flash_breadcrumbs: #182152; // breadcrumbs ("Continuities » Continuity » Section » Thread")
+$font_color_flash_bold: #F3F3F3;
+
+.flash.breadcrumbs {
+  background-color: $bg_color_flash_breadcrumbs;
+  &, a, a:visited { color: $font_color_link_visited; }
+  b { color: $font_color_flash_bold; }
 }
+.flash.success { background-color: $bg_color_flash_success; }
+.flash.error { background-color: $bg_color_flash_error; }
+.flash.inbox { background-color: $bg_color_flash_inbox; }
 
-$EVEN_COLOR: #E3E3E3;
-$ODD_COLOR: #D8D8D8;
+/* - main page */
+$bg_color_page: #6C697C; // overall page
+$font_color_page: #DADADA;
 
-#post-menu-box {
-  border-top: 1px solid #808080;
-  border-top: 1px solid rgba(128,128,128,0.5);
-  background-color: #BCBCBC;
-  color: #101010;
+body { background-color: $bg_color_page; color: $font_color_page; }
 
-  a { color: inherit; }
-  a:hover div { background-color: #D3D3D3; }
-  div {
-    border-bottom: 1px solid #808080;
-    border-bottom: 1px solid rgba(128,128,128,0.5);
-  }
+$bg_color_spacer_alt: #182152;
+$font_color_search_box: #FEFEFE;
+$bg_color_search_box: #182152;
+
+.search-box { background-color: $bg_color_search_box; color: $font_color_search_box; }
+.spacer-alt { background-color: $bg_color_spacer_alt; }
+
+/* - misc */
+$bg_color_descriptions: #484357; // description subheadings on templates page ("single-description"), *not* posts
+$font_color_descriptions: #DADADA;
+$font_color_descriptions_link: #B5AFCB;
+.single-description {
+  background-color: $bg_color_descriptions;
+  color: $font_color_descriptions;
+  a { color: $font_color_descriptions_link; }
+  a:visited { color: $font_color_link_visited; }
 }
+
+/* - content headers */
+$bg_color_head: #13132F; // primary content header
+$font_color_head_link: #B5AFCB;
+$bg_color_sub: #182152; // secondary content header
+$bg_color_subber: #484357; // tertiary content header
+$bg_color_ender: #13132F; // form table footer with submit
+$font_color_ender: #F3F3F3;
+$font_color_sub: #F3F3F3;
+$font_color_subber: #DADADA;
+$font_color_subber_bold: #F3F3F3;
+
+th, .content-header {
+  background-color: $bg_color_head;
+  a { color: $font_color_head_link; }
+  a:visited { color: $font_color_link_visited }
+}
+.sub { color: $font_color_sub; background-color: $bg_color_sub; }
+
+.form-table-ender { background-color: $bg_color_ender; color: $font_color_ender; }
+.subber {
+  background-color: $bg_color_subber;
+  color: $font_color_subber;
+  &, a, a:visited { color: $font_color_link_visited; }
+  b { color: $font_color_subber_bold; }
+}
+
+/* - content */
+$bg_color_even: #31323B;
+$bg_color_odd: #25262E;
```

Comment:
even and odd should be `#E3E3E3` and `#D8D8D8` respectively (think you copied this wrong?)

Reply:
looks like I copied it from starrydark